client/src/App.js

```diff
@@ -1,8 +1,25 @@
 import './App.css';
 
-import React from 'react';
+import React, { useState } from 'react';
+
+import PageNav from './components/PageNav/PageNav';
 import Search from './components/Search/Search';
+import Visualization from './components/Visualization/Visualization';
+
+const App = () => {
+  const [currentPage, setCurrentPage] = useState('Ads Search');
+
+  const pageMap = {
+    'Ads Search': <Search />,
+    'Data Visualization': <Visualization />,
+  };
 
-const App = () => <Search />;
+  return (
+    <div>
+      <PageNav currentPage={currentPage} setCurrentPage={setCurrentPage} />
```

Comment:
what is the difference between `currentPage` and `setCurrentPage`?

Reply:
`currentPage` is the value and `setCurrentPage` is the function that sets the value of `currentPage`